server/src/internalClusterTest/java/org/opensearch/search/aggregations/bucket/NaNSortingIT.java

```diff
@@ -171,6 +171,7 @@ public void setupSuiteScopeCluster() throws Exception {
             }
             client().prepareIndex("idx").setSource(source.endObject()).get();
         }
+        indexRandomForMultipleSlices("idx");
         refresh();
```

Comment:
should we remove this refresh call here?

Reply:
I know it's redundant know but I still would prefer to keep it in case the `indexRandomForMultipleSlices` implementation changes in the future.

Reply:
I would then call the function after refresh

Reply:
Changed